package/com.unity.formats.usd/Dependencies/USD.NET/serialization/Primvar.cs

```diff
@@ -142,6 +142,10 @@ public Type GetValueType()
             return typeof(T);
         }
 
+        public bool IsArray => typeof(T).IsArray;
+
+        public int Length => IsArray && value != null ? (value as Array).Length : 0;
```

Comment:
Nit pick, but I would add parens around (IsArray && value != null) for readability.

Reply:
Fixed in dd63d81